ui/raidboss/data/06-ew/ultimate/dragonsongs_reprise_ultimate.txt

```diff
@@ -89,8 +91,95 @@ hideall "--sync--"
 601.5 "Holy Bladedance" sync / 1[56]:[^:]*:Ser Adelphel:62D2:/
 602.5 "--sync--" sync / 1[56]:[^:]*:King Thordan:63C4:/
 602.8 "Conviction" sync / 1[56]:[^:]*:Ser Hermenost:63DF:/
+
+604.9 "--targetable--"
 611.0 "Ancient Quaga" sync / 1[56]:[^:]*:King Thordan:63C6:/
 621.2 "Heavenly Heel" sync / 1[56]:[^:]*:King Thordan:63C7:/
-624.3 "Ascalon's Might" #sync / 1[56]:[^:]*:King Thordan:63C5:/
-626.1 "Ascalon's Might" #sync / 1[56]:[^:]*:King Thordan:63C5:/
-630.4 "--sync--" sync / 1[56]:[^:]*:King Thordan:63C4:/
+624.4 "Ascalon's Might 1" #sync / 1[56]:[^:]*:King Thordan:63C5:/
+626.0 "Ascalon's Might 2" #sync / 1[56]:[^:]*:King Thordan:63C5:/
+627.7 "Ascalon's Might 3" #sync / 1[56]:[^:]*:King Thordan:63C5:/
+
+630.5 "--sync--" sync / 1[56]:[^:]*:King Thordan:63C4:/
+631.6 "Knights of the Round" sync / 1[56]:[^:]*:King Thordan:63ED:/
+638.7 "Sanctity of the Ward" sync / 1[56]:[^:]*:King Thordan:63E1:/
+641.8 "--untargetable--"
+652.2 "Sacred Sever x4" duration 5.3 #sync / 1[56]:[^:]*:Ser Zephirin:63E3:/
+652.4 "The Dragon's Glory" sync / 1[56]:[^:]*:King Thordan:63D2:/
+652.4 "The Dragon's Gaze" sync / 1[56]:[^:]*:King Thordan:63D1:/
+658.5 "--sync--" sync / 1[56]:[^:]*:King Thordan:63C4:/
+669.8 "Heavens' Stake" sync / 1[56]:[^:]*:Ser Charibert:6FAE:/
+669.8 "Hiemal Storm" sync / 1[56]:[^:]*:Ser Haumeric:63E6:/
+671.9 "Conviction 1" sync / 1[56]:[^:]*:Ser Hermenost:737B:/
+674.7 "Holy Comet x7" duration 12.7 sync / 1[56]:[^:]*:Ser Noudenet:63E8:/
+685.1 "Faith Unmoving" sync / 1[56]:[^:]*:Ser Grinnaux:62DC:/
+685.2 "Conviction 2" sync / 1[56]:[^:]*:Ser Hermenost:6FEA:/
+692.5 "--targetable--"
+
+692.6 "--sync--" sync / 1[56]:[^:]*:King Thordan:63BC:/
+697.7 "--sync--" sync / 1[56]:[^:]*:King Thordan:63BD:/ window 20,20
+706.1 "Ultimate End" sync / 1[56]:[^:]*:King Thordan:63BE:/
+
+711.8 "--sync--" sync / 1[56]:[^:]*:King Thordan:63BF:/
+718.4 "Broad Swing 1" sync / 1[56]:[^:]*:King Thordan:63C[01]:/
+726.9 "Broad Swing 2" sync / 1[56]:[^:]*:King Thordan:63C[01]:/
+732.0 "--sync--" sync / 14:[^:]*:King Thordan:63C3:/ window 30,10
+738.0 "Aetheric Burst Enrage" sync / 1[56]:[^:]*:King Thordan:63C3:/
+
+
+### Phase 3: Nidhogg
+# Tower ability notes:
+# 670E = Dark High Jump (circle marker)
+# 670F = Dark Spineshatter Dive (upward/forwards arrow)
+# 6710 = Dark Elusive Jump (downward/backwards arrow)
+# 6711, 6717, 6178, 6719, 671B = Darkdragon Dive (taking a tower, unsure why different)
```

Comment:
From looking at a bunch of logs, the Darkdragon Dive IDs do the following:

`6711` are the single soak towers that spawn from placing `670E` (High), `670F` (Spineshatter), and `6710` (Elusive) jumps.

`6717`, `6718`, `6719` are towers that spawn during the [4x Towers](https://github.com/quisquous/cactbot/blob/be2d076fcfba51df4ddf7c7f4c7edee7cab00d79/ui/raidboss/data/06-ew/ultimate/dragonsongs_reprise_ultimate.txt#L162) segment.

- `6717` is a single soak tower
- `6718` is a 2 person soak tower
- `6719` is a 3 person soak tower

`671B` happens whenever the party fails to soak a tower. (Tower is left empty.)

Reply:
Thanks!